flask_restless/helpers.py

```diff
@@ -622,3 +650,35 @@ def __call__(self, model, _apimanager=None, **kw):
 #:     <class 'mymodels.Person'>
 #:
 model_for = ModelFinder()
+
+#: Returns the primary_key to be used for the given model, as specified
+#: by the ``primary_key`` keyword argument to :meth:`APIManager.create_api`
+#: when it was previously invoked on the model.
+#:
+#: `primary_key` is a string corresponding to the primary key identifier
+#: to be used by flask-restless for a model. If no primary key has been set
+#: at the flask-restless level (by using the ``primary_key`` keyword argument
+#: when calling :meth:`APIManager.create_api_blueprint`, the model's primary
+#: key will be returned. If no API has been created for the model, this
+#: function raises a `ValueError`.
+#:
+#: If `_apimanager` is not ``None``, it must be an instance of
+#: :class:`APIManager`. Restrict our search for endpoints exposing `model` to
+#: only endpoints created by the specified :class:`APIManager` instance.
+#:
+#: For example, suppose you have a model class ``Person`` and have created the
+#: appropriate Flask application and SQLAlchemy session::
+#:
+#:     >>> from mymodels import Person
+#:     >>> manager = APIManager(app, session=session)
+#:     >>> manager.create_api(Person, primary_key='name')
+#:     >>> primary_key_for('people')
+#:     'name'
+#:     >>>
+#:     >>> # This is in contrast to the typical default:
```

Comment:
This comment should be prose, not part of the code, i.e.

        >>> primary_key_for('people')
        'name'

    This is in contrast to the typical default:

        >>> manager = APIManager(app, session=session)